px-map-behavior-marker.es6.js

```diff
@@ -345,7 +342,12 @@
       return {
         type: this.type,
         symbol: this.symbolClass,
-        styleScope: this.isShadyScoped() ? this.getShadyScope() : undefined
+        icon: this.icon,
+        styleScope: this.isShadyScoped() ? this.getShadyScope() : undefined,
```

Comment:
can we store the getComputedStyleValue in a var and use that to avoid calling it 3 times